package_python_function/packager.py

```diff
@@ -1,23 +1,18 @@
 from __future__ import annotations
 
 import logging
-import os
 import shutil
-import time
-import zipfile
 from pathlib import Path
 from tempfile import NamedTemporaryFile
-from typing import TYPE_CHECKING
+from zipfile import ZIP_DEFLATED, ZIP_STORED
 
 from .python_project import PythonProject
-
-if TYPE_CHECKING:
-    from typing import Tuple
+from .reproducible_zipfile import ZipFile
 
 logger = logging.getLogger(__name__)
 
 class Packager:
-    AWS_LAMBDA_MAX_UNZIP_SIZE = 262144000
+    AWS_LAMBDA_MAX_UNZIP_SIZE = 262_144_000
```

Comment:
Holy moly I didn't know you could do this.  TIL about PEP-515.  These days it is very rare for me to learn something new while reviewing a PR. Thanks!